addon/utils/sparql-helpers.ts

```diff
@@ -1,13 +1,25 @@
 import * as RDF from '@rdfjs/types';
+import { optionMapOr } from '@lblod/ember-rdfa-editor-lblod-plugins/utils/option';
 
-export type QueryResult = {
+export type QueryResult<Binding = Record<string, RDF.Term>> = {
```

Comment:
this is so minor that I don't expect you to change it, but I tend to prefer `interface` when possible, for no real reason other than consistency